mockgen/mockgen.go

```diff
@@ -305,14 +305,14 @@ func (g *generator) Generate(pkg *model.Package, pkgName string, outputPackagePa
 	g.p("")
 	g.p("import (")
 	g.in()
-	for path, pkg := range g.packageMap {
-		if path == outputPackagePath {
+	for pkgPath, pkg := range g.packageMap {
```

Comment:
We import `path` so we should not name variables with the same name as the import.